modules/core/src/views/view.ts

```diff
@@ -53,7 +53,7 @@ export default abstract class View<
 > {
   id: string;
   abstract get ViewportType(): ConstructorOf<Viewport>;
-  abstract get ControllerType(): ConstructorOf<Controller<any>>;
+  protected abstract get ControllerType(): ConstructorOf<Controller<any>>;
```

Comment:
Added to prevent similar bugs in the future. As far as I can tell `ControllerType` is only used as a default value for `this.controller.type`